twake/frontend/src/app/services/RouterService.ts

```diff
@@ -118,6 +130,11 @@ class RouterServices extends Observable {
     });
   }
 
+  getRoutes(auth?: 'console' | 'internal'): RouteType[] {
+    const loginRoute = auth && auth === 'console' ? this.consoleLogin : this.internalLogin;
+    return [...[loginRoute], ...this.routes];
+  }
+
```

Comment:
Why not having a parent component like "Login" that will return "ConsoleLogin" or "InternalLogin" instead of this route system ? It will avoid to have a "auth" parameter here too.

Reply:
Yes, I also plan to do the same for logout

Reply:
(and /logout still not exist yet)

Reply:
I did something with React.lazy to save some resources https://github.com/linagora/Twake/pull/1657/commits/afee8b060dfea50f97ea1cd33b3d687f3108d469